src/main/java/at/hannibal2/skyhanni/config/features/gui/customscoreboard/PartyConfig.java

```diff
@@ -4,21 +4,25 @@
 import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorBoolean;
 import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorSlider;
 import io.github.notenoughupdates.moulconfig.annotations.ConfigOption;
-import io.github.notenoughupdates.moulconfig.observer.Property;
 
 public class PartyConfig {
     @Expose
     @ConfigOption(name = "Max Party List", desc = "Max number of party members to show in the party list (you are not included).")
     @ConfigEditorSlider(
         minValue = 1,
-        maxValue = 25, // why do I even set it so high
+        maxValue = 25,
         minStep = 1
     )
-    public Property<Integer> maxPartyList = Property.of(4);
+    public Integer maxPartyList = 4;
```

Comment:
why is this no longer a  observer? this is counter productive to the plan of having the feature react to changes and not periodically udpate everything